### DISCUSSION TITLE
Раздел с подготовкой к собесам

### DISCUSSION THREAD

**Discussion by kxnkxv** [doka-guide/content#3323] [Идеи] [answered]
Добавить раздел с разбором популярных вопросов с собеседований

**Comment by solarrust** [comment]
Привет! 
Хорошая, нужная тема. Но, кажется, она сейчас немного не встраивается в идею справочника. У нас прямо сейчас есть классный формат «На практике», где можно делиться своим опытом по использованию технологии. Например, написать, что вот про это часто спрашивают на собеседовании и отвечать на это надо вот так и вот так.
Когда таких советов станет больше, то можно будет вернуться к обсуждению отдельного раздела под это дело. 
У тебя уже есть идеи, о каком вопросе можно рассказать уже сейчас?

**Reply by kxnkxv** [reply]
Кажется, что она как-раз таки встраивается в идею справочника + это способ усилить связи внутри между статьями
Идешь изучать что такое event loop -> видишь что это часто спрашивают на собесах -> идешь узнавать что спрашивают и как отвечать. И в обратном порядке.

**Reply by pa7lux** [reply, marked as answer]
Возможно, рубрика "На собесе" применима ко многим статьям, и нам стоит задуматься не о разделе, а о блоке внутри конкретной темы, куда контрибьютеры смогут принести "Вопрос и ответ с собеса про Event Loop". Если рубрика начнет разрастаться, можно смапить это в раздел
